vision.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  vision_team = [
+    "ahcorde",
+    "mikeferguson",
```

Comment:
He hasn't been active in a while - but should we also include JWhitleyWork?

Reply:
It's easy enough to add him in a follow-up PR as needed. Let's keep the team limited to active contributors for the time being.